package.json

```diff
@@ -23,25 +23,27 @@
     "dist/*.map"
   ],
   "scripts": {
-    "prepare": "yarn make-release && yarn build && mocha ./dist/quickjs.test.js && yarn tarball && yarn smoketest-node",
+    "prepare": "yarn prepack && mocha ./dist/quickjs.test.js && yarn tarball && yarn smoketest-node",
     "update-quickjs": "git subtree pull --prefix=quickjs --squash git@github.com:ldarren/QuickJS.git mod",
     "make-debug": "make DEBUG=1 -j 8",
-    "make-release": "which emcc && make clean && make -j 8",
+    "make-release": "embin which emcc && make clean && make -j 8",
     "clean": "make clean",
     "build": "tsc",
     "doc": "typedoc --plugin typedoc-plugin-markdown ts",
     "test": "yarn make-debug && mocha",
     "prettier": "prettier --write ./*.json ./**/*.ts* ./*.js* ./*.md",
     "prettier-check": "prettier --check ./*.json ./**/*.ts* ./*.js* ./*.md",
     "run-n": "yarn make-debug && ./build/wrapper/native/test.exe",
+    "prepack": "yarn make-release && yarn build",
     "tarball": "mkdir -p build && yarn pack --filename build/quickjs-emscripten.tgz",
-    "smoketest-node": "cd examples/typescript-smoketest && yarn add ../../build/quickjs-emscripten.tgz --update-checksums && tsc --lib ES2015,dom index.ts && node index.js",
-    "smoketest-cra": "cd examples/website && yarn add ../../build/quickjs-emscripten.tgz --update-checksums && yarn build"
+    "smoketest-node": "cd examples/typescript-smoketest && YARN_CHECKSUM_BEHAVIOR=update yarn add quickjs-emscripten@../../build/quickjs-emscripten.tgz && tsc --lib ES2015,dom index.ts && node index.js",
+    "smoketest-cra": "cd examples/website && YARN_CHECKSUM_BEHAVIOR=update yarn add quickjs-emscripten@../../build/quickjs-emscripten.tgz && yarn build"
   },
   "devDependencies": {
     "@types/emscripten": "^1.38.0",
     "@types/mocha": "^5.2.7",
     "@types/node": "^13.1.4",
+    "embin-linux": "1.39.19-4",
```

Comment:
Is there a macOS or Darwin variant? I develop on Mac

Reply:
Yep, adding `embin-darwin` should make it work on both systems. I'll do that and mirror the CI job to be sure.